packages/fxa-auth-server/lib/senders/renderer/index.ts

```diff
@@ -63,51 +64,27 @@ class Renderer extends Localizer {
 
     if (template !== '_storybook') {
       /*
-       * TODO: finish pulling subject & actions out of mailer and into template files, then
-       * this 'if' can be removed. FXA-4623
-       *
        * 'Subject' and 'action' must be localized BEFORE the email is rendered because:
        * 1) These values are needed in layout files and aren't easily localized, since
        * `subject` goes inside `mj-title` and `action` goes in a script in `metadata.mjml`
        * 2) We need to return a localized `subject` back to the mailer
        */
-      if (
-        template === 'verify' ||
-        template === 'postRemoveTwoStepAuthentication' ||
-        template === 'verifyLoginCode' ||
-        template === 'downloadSubscription'
-      ) {
-        const { subject, action } = JSON.parse(
-          await this.bindings.fetchResource(
-            `${this.bindings.opts.templates.basePath}/templates/${template}/includes.json`
-          )
-        ) as unknown as GlobalTemplateValues;
-
-        if (action) {
-          const [localizedSubject, localizedAction] = await Promise.all([
-            this.localizeAndRender(l10n, subject, context),
-            this.localizeAndRender(l10n, action, context),
-          ]);
-
-          context.subject = localizedSubject;
-          context.action = localizedAction;
-        } else {
-          context.subject = await this.localizeAndRender(
-            l10n,
-            subject,
-            context
-          );
-        }
+      const { subject, action } = await this.getGlobalTemplateValues(context);
```

Comment:
Perhaps add some logging or fail hard if subject is missing? Not 100% sure this is needed, but it might be an extra safeguard.